### PR TITLE
Adding UK About pages to menus

### PR DESCRIPTION
**Pull request by lin-d-hop**
#### What? Why?

UK are adding a blog site for news, stories, videos and how tos etc etc.

#### What should we test?

Please test that the menu structure works correctly, with all menu links functioning as you expect. Please also test on mobile.

To test this properly some server changes also need to be made so when you are ready to test please:
1) Test the menus havent changed after deploying
2) Ask Lynne to make the server changes and test again.

#### Release notes

No release notes as this is UK specific

### CODE REVIEW

File: app/views/shared/menu/_large_menu.html.haml [hunk 1]
```diff
@@ -26,6 +26,23 @@
           %a{href: "https://openfoodnetwork.org/au/learn/"}
             %span.nav-primary
               = t 'label_learn'
+      - elsif feature? :about_blog_homepage
```
Comment (sauloperez) [inline app/views/shared/menu/_large_menu.html.haml]: Is this some kind of feature flags implementation? It's the first time I see it

Comment (lin-d-hop) [reply app/views/shared/menu/_large_menu.html.haml]: Yep, Aus use it. On the server also is added in config/initializers/feature_toggle.rb:
require 'open_food_network/feature_toggle'

module OpenFoodNetwork
  FeatureToggle.class_eval do
    private

    def self.features
      {order_cycles: true,
       about_blog_homepage: true}
    end
  end
end

Comment (lin-d-hop) [reply app/views/shared/menu/_large_menu.html.haml]: Thanks to Maikel for that tip :-)

File: .travis.yml [hunk 2]
```diff
@@ -22,8 +22,10 @@ env:
     - CI_NODE_INDEX=3
     - CI_NODE_INDEX=4 KARMA="true" GITHUB_DEPLOY="true"
 
-before_install:
-  - ./script/upgrade_bundler.sh
+# Disabling due to travis errors.
+# See @oeoeaio d46888b
+#before_install:
+#  - ./script/upgrade_bundler.sh
```
Comment (sauloperez) [inline .travis.yml]: I'd rather remove the lines. That is what commits and their descriptions are for IMO.

Comment (lin-d-hop) [reply .travis.yml]: I'd rather merge rob's commit and remove this commit from this PR altogether.
I should add the 'wip' label :-)

**Comment by lin-d-hop** [comment]
@OliverUK tested the menu items after deploy but without the initialiser and the menus don't change - both on mobile and desktop
After adding initialiser menus work as expected.

**Comment by lin-d-hop** [comment]
Tis a minor change. I'd like to merge if other devs are happy?

**Comment by daniellemoorhead** [comment]
@lin-d-hop Rob had a quick look and is happy for you to merge this. Go right ahead :)

#remembertoaddthemilestone 😃

**Pull request merged**